definitions/tools/replace_vcf_sample_name.cwl

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "Replace the sample name in a gzipped vcf and return"
+baseCommand: ['/bin/bash', 'replacer.sh']
+requirements:
+    - class: ShellCommandRequirement
+    - class: DockerRequirement
+      dockerPull: "mgibio/samtools-cwl:1.0.0"
+    - class: ResourceRequirement
+      ramMin: 8000
+    - class: InitialWorkDirRequirement
+      listing:
+      - entryname: 'replacer.sh'
+        entry: |
+            set -eou pipefail
+            zcat $1 | sed -e "s/$2/$3/g" | /opt/htslib/bin/bgzip
```

Comment:
```suggestion
            if [[ "$1" =~ ".vcf.gz" ]];then
                zcat "$1" | sed -e "s/$2/$3/g" | /opt/htslib/bin/bgzip >renamed.vcf.gz
            else
                cat "$1" | sed -e "s/$2/$3/g" | /opt/htslib/bin/bgzip >renamed.vcf.gz
            fi
            tabix renamed.vcf.gz
```

Reply:
would also need to update the glob (or pass in the output file name), but you get the gist